### PR TITLE
[DAT-496] - Change the "Province/State" field to use "Select"

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Change the "Province/State" field to use "Select" instead of "Text"

**Before:**

image

**After:**

image

**Task:** [DAT-496](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-496)

### CODE REVIEW

**Comment by jcamposmk** [comment]
:+1:

File: src/services/static-data-client.ts [hunk 1]
```diff
@@ -26,4 +27,31 @@ export class HttpStaticDataClient implements StaticDataClient {
       return { success: false, error: error };
     }
   }
+
+  public async getStatesData(country: string, language: string): Promise<any> {
+    try {
+      const response = await this.axios.request({
+        method: 'GET',
+        url: this.baseUrl + `/${language === 'es' ? 'states-es-v1' : 'states-en-v1'}.json`,
+        headers: { 'Cache-Control': 'no-store,max-age=0' },
```
Comment (cbernat) [inline src/services/static-data-client.ts:36]: As we talked this should not be neccesary right now, we will see in the future.

**Comment by cbernat** [comment]
👍

File: src/services/static-data-client.ts (hunk 1 above)
Comment (jhoffman-ms) [reply src/services/static-data-client.ts]: Done

**Pull request merged**